ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml

```diff
@@ -0,0 +1,260 @@
+                           bool infiniteSupply,
+                           bool transferable,
+                           bool selfDestruct,
+                           int chainId)
+
+    QtObject {
+        id: d
+
+        readonly property bool isFullyFilled: d.artworkSource.toString().length > 0
+                                              && !!nameInput.text
+                                              && !!symbolInput.text
+                                              && !!descriptionInput.text
+                                              && (unlimitedSupplyItem.checked || (!unlimitedSupplyItem.checked && supplyInput.text.length > 0))
+
+
+        readonly property int imageSelectorRectWidth: 280
+
+        property url artworkSource
+        property int networkSelected
+    }
+
+    contentWidth: mainLayout.width
+    contentHeight: mainLayout.height
+    padding: 0
+
+    ColumnLayout {
+        id: mainLayout
+
+        width: root.viewWidth
+        spacing: Style.current.padding
+
+        StatusImageSelector {
+            Layout.preferredHeight: d.imageSelectorRectWidth + 34 /*Label*/
+            Layout.preferredWidth: d.imageSelectorRectWidth + 8 /*Add button*/
+            labelText: qsTr("Artwork")
+            uploadText: qsTr("Drag and Drop or Upload Artwork")
+            additionalText: qsTr("Images only")
+
+            onFileSelected: d.artworkSource = file
+        }
+
+        component CustomStatusInput: StatusInput {
+            id: customInput
+
+            property string errorText
+
+            Layout.fillWidth: true
+            validators: [
+                StatusMinLengthValidator {
+                    minLength: 1
+                    errorMessage: Utils.getErrorMessage(root.errors,
+                                                        customInput.errorText)
+                },
+                StatusRegularExpressionValidator {
+                    regularExpression: Constants.regularExpressions.alphanumericalExpanded
+                    errorMessage: Constants.errorMessages.alphanumericalExpandedRegExp
+                }
+            ]
+        }
+
+        CustomStatusInput {
+            id: nameInput
+
+            label: qsTr("Name")
+            charLimit: 30
+            placeholderText: qsTr("Name")
+            errorText: qsTr("Collectible name")
+        }
+
+        CustomStatusInput {
+            id: descriptionInput
+
+            label: qsTr("Description")
+            charLimit: 280
+            placeholderText: qsTr("Describe your collectible")
+            input.multiline: true
+            input.verticalAlignment: Qt.AlignTop
+            input.placeholder.verticalAlignment: Qt.AlignTop
+            minimumHeight: 108
+            maximumHeight: minimumHeight
+            errorText: qsTr("Collectible description")
+        }
+
+        CustomStatusInput {
+            id: symbolInput
+
+            label: qsTr("Token symbol")
+            placeholderText: qsTr("CK")
```

Comment:
CK?

Reply:
I've asked to the designers. In figma it is like this but.. just in case.. Waiting their answer!

Reply:
I will modify it to: "Three letter token abbreviation e.g. ABC" as agreed with designers!